sky/skylet/job_lib.py

```diff
@@ -182,13 +182,18 @@ def _run_job(self, job_id: int, run_cmd: str):
 
     def schedule_step(self, force_update_jobs: bool = False) -> None:
         jobs = self._get_jobs()
-        if len(jobs) > 0 or force_update_jobs:
+        if force_update_jobs:
             update_status()
```

Comment:
Can be moved above L184?

Reply:
Moved. It does not matter though, as the `get_jobs` above is just getting jobs from the pending table. I renamed the function to make it more readable.